packages/react-components/react-positioning/src/types.ts

```diff
@@ -263,3 +263,21 @@ export type PositioningShorthandValue =
   | 'after-bottom';
 
 export type PositioningShorthand = PositioningProps | PositioningShorthandValue;
+
+// ---
+
+export type PositioningConfigurationFnOptions = Omit<
+  PositioningOptions,
+  | 'disableUpdateOnResize'
```

Comment:
why is `disableUpdateOnResize` omitted?

Reply:
I don't think that it should be, you're right. Updated